src/index.js

```diff
@@ -0,0 +1,7 @@
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
```

Comment:
babel-preset is usually pretty good at injecting these itself based on the compiled code - was there a reason you included these here?

Reply:
I couldn't get it working as a module if I didn't import them explicitly. 

As a test, I added a dependency to Charts' package.json pointing directly to the branch
```
dependencies: {
  ...
  "expression-to-mql": "mongodb-js/expression-to-mql#all-the-files",
  ...
}
```

Without those explicit imports I got an error, presumably because the module mode would require the `./src` directly, and not the babelified code from `./dist`. I'm not an expert in that area, if there's a better way to set it up I'd welcome suggestions.

Reply:
Have a look at https://survivejs.com/maintenance/packaging/building/#configuring-babel-for-tree-shaking

Reply:
This is now done, thanks for the link.